### ISSUE TITLE
Contradictory HACS installation instructions for folding module

### ISSUE THREAD

**Issue by haakonstorm** [thomasloven/lovelace-fold-entity-row#67]
My Home Assistant version: 0.100.2

My lovelace configuration method (GUI or yaml):
YAML

What I am doing:
- Trying to install the folding module

What I expected to happen:
- That the instructions provided via HACS for installing would end up in a working folding module

What happened instead:
- Invalid HA config found

**Minimal** steps to reproduce:
- Two methods are contradictory:

Readme text:
```
resources:
  url: /local/fold-entity-row.js
  type: module
Usage
(...)
```

HACS text:
```

When you add this to your configuration use this: 
  - url: /community_plugin/lovelace-fold-entity-row/fold-entity-row.js
    type: module 
```

```yaml
# The least ammount of code possible to reproduce my error

  url: /local/fold-entity-row.js
  type: module


# End of code
```

Error messages from the browser console:

---

**By putting an X in the boxes ([ ]) below, I indicate that I:**

[X] Understand that this is a channel for reporting bugs, not a support forum (https://community.home-assistant.io/).

[X] Have made sure I am using the latest version of the plugin.

[X] Have followed the troubleshooting steps of the "Common Problems" section of https://github.com/thomasloven/hass-config/wiki/Lovelace-Plugins.

[X] Understand that leaving one or more boxes unticked or failure to follow the template above may increase the time required to handle my bug-report, or cause it to be closed without further action.

**Comment by haakonstorm** [comment]
Hey, that was fast, but I was referencing to the two different paths to the `.js` - this might better be asked over at `HACS` but I'm having trouble getting `HA` to actually find and use  the module. 
- For `HASSIO` and `HACS` installation, should we use:
  -  `- url: /community_plugin/lovelace-fold-entity-row/fold-entity-row.js`
  - or
  - `url: /local/fold-entity-row.js` ?

Intuitively, I understand `/local/` should be used for standalone i.e. non-hacs installations, and `/community_plugin/` for HACS, but either way I'm having trouble. At one point it was working in all browsers I tested on two macs (one old `MBP15` and a new `iMac Pro` both running latest `Catalina` - but the folding js would not work ("not found") on either iOS13 Home Assistant Companion 2.0β, and similarly on iPadOS13 2.0β , or the iOS13 or iPadOS13 1.xx non-beta versions. I just realized I might have a corrupted SD-Card on my hassio install, since my HASS now experiences groundhog day (any changes, upgrades, addon deletions whatnot are reverted upon boot of hassio host). 
- Running on latest hassio on `Raspberry Pi 4`.
